clicache/integration-test/BuiltinCacheableWrappersN.cs

```diff
@@ -270,6 +271,7 @@ public static void RegisterBuiltinsJavaHashCode(long dateTime)
         typeof(CacheableHugeUnicodeStringWrapper), CacheableHugeUnicodeStringWrapper.Create);
 
       //need to register pdx type
+      Assert.NotNull(CacheHelper.DCache);
```

Comment:
Why are we asserting framework variables? Asserts should be reserved for things that are part of the test qualifications. Exceptions should be thrown for things that fail as a result of tooling issues.

Reply:
This was us trying to figure out why some of the tests marked `omitted` were failing.  We found out that some of those tests were failing because `DCache` was null.  Since it was non-obvious that this was the cause we left this statement in to help in future for fixing the tests still `omitted`

Reply:
:/